src/d/actor/d_a_tag_attention.cpp

```diff
@@ -6,8 +6,10 @@
 #include "d/actor/d_a_tag_attention.h"
 #include "d/d_procname.h"
 #include "d/d_priority.h"
-#include "d/d_cc_d.h"
+#include "f_op/f_op_actor_mng.h"
+#include "d/d_com_inf_game.h"
 
+/* ram:8049162C-ram:804917AF  */
```

Comment:
RELs don't get loaded at a fixed RAM address. These addresses are just for our Ghidra project and aren't useful in the game itself
```suggestion
```